src/org/zaproxy/zap/extension/quickstart/launch/ExtensionQuickStartLaunch.java

```diff
@@ -0,0 +1,345 @@
+                    .getString("quickstart.button.tooltip.launch"));
+
+            launchButton.addActionListener(new java.awt.event.ActionListener() {
+                @Override
+                public void actionPerformed(java.awt.event.ActionEvent e) {
+                    new Thread(new Runnable() {
+                        @Override
+                        public void run() {
+                            try {
+                                WebDriver wd = getExtSelenium()
+                                        .getProxiedBrowserByName(
+                                                getBrowserComboBox()
+                                                        .getSelectedItem()
+                                                        .toString());
+                                if (wd != null) {
+                                    if (getQuickStartLaunchParam()
+                                            .isZapStartPage()) {
+                                        wd.get(API
+                                                .getInstance()
+                                                .getBaseURL(
+                                                        API.Format.OTHER,
+                                                        QuickStartLaunchAPI.API_PREFIX,
+                                                        API.RequestType.other,
+                                                        QuickStartLaunchAPI.OTHER_START_PAGE,
+                                                        true));
+
+                                    } else if (!getQuickStartLaunchParam()
+                                            .isBlankStartPage()) {
+                                        wd.get(getQuickStartLaunchParam()
+                                                .getStartPage());
+                                    }
+                                    // Use the same browser next time, as long
+                                    // as it worked
+                                    getQuickStartLaunchParam()
+                                            .setDefaultBrowser(
+                                                    getBrowserComboBox()
+                                                            .getSelectedItem()
+                                                            .toString());
+                                }
+                            } catch (Exception e1) {
+                                LOGGER.error(e1.getMessage(), e1);
+                            }
+                        }
+                    }).start();
+                }
+            });
+        }
+        return launchButton;
+    }
+
+    private JComboBox<Object> getBrowserComboBox() {
+        if (browserComboBox == null) {
+            browserComboBox = new JComboBox<Object>();
+            try {
+                for (String browser : getExtSelenium()
+                        .getUsableProvidedBrowserUINameList()) {

```

Comment:
`ExtensionSelenium` could be changed to provide the combo box model with configured browsers, to always have the latest browsers available.